projects/js-packages/publicize-components/src/components/global-modals/index.tsx

```diff
@@ -1,8 +1,15 @@
+import { siteHasFeature } from '@automattic/jetpack-script-data';
 import { getSocialScriptData } from '../../utils/script-data';
 import { ThemedShareStatusModal as ShareStatusModal } from '../share-status';
 
 export const GlobalModals = () => {
 	const { feature_flags } = getSocialScriptData();
 
-	return <>{ feature_flags.useShareStatus ? <ShareStatusModal /> : null }</>;
+	return (
+		<>
+			{ feature_flags.useShareStatus || siteHasFeature( 'social-share-status' ) ? (
```

Comment:
When I add the blog sticker, the feature gets added to the features array for self hosted sites. It appears to work as is, why do we need the `siteHasFeature( 'social-share-status' )` part here?

Reply:
For self hosted sites, yes, but not for WPCOM sites.